draft-ietf-privacypass-auth-scheme.md

```diff
@@ -172,13 +172,13 @@ origin names, they are delimited with commas "," without any whitespace.
 When used in an authentication challenge, the "PrivateToken" scheme uses the
 following attributes:
 
-- "challenge", which contains a base64url-encoded {{!RFC4648}} TokenChallenge
-value. Since the length of the challenge is not fixed, the base64url data MUST
-include padding. This MUST be unique for every 401 HTTP response to prevent
+- "challenge", which is a quoted string containing a base64url-encoded {{!RFC4648}}
+TokenChallenge value. Since the length of the challenge is not fixed, the base64url
+data MUST include padding. This MUST be unique for every 401 HTTP response to prevent
 replay attacks. This attribute is required for all challenges.
 
-- "token-key", which contains a base64url encoding of the public key for
-use with the issuance protocol indicated by the challenge. Since the length of
+- "token-key", which is a quoted string containing a base64url encoding of the public
+key for use with the issuance protocol indicated by the challenge. Since the length of
 the key is not fixed, the base64url data MUST include padding. This attribute MAY
 be omitted in deployments where clients are able to retrieve the issuer key using
 an out-of-band mechanism.
```

Comment:
```suggestion
- "token-key", which contains a base64url encoding of the public key for
use with the issuance protocol indicated by the challenge. Since the length of
the key is not fixed, the base64url value MUST include padding. As an
Authentication Parameter (`auth-param` from {{!RFC9110}}), the value can be
either a token or a quoted-string, depending on if the base64url string
includes "=" characters. This attribute MAY be omitted in deployments where
clients are able to retrieve the issuer key using an out-of-band mechanism.
```

Reply:
(same type of comment here; the recipient needs to handle both formats no matter what)

Reply:
Updated!